### PR TITLE
version: refactor help text

### PR DESCRIPTION
**Pull request by skoh7645**
Refactor help text for `appsody version`

Fixes: #588

### CODE REVIEW

File: cmd/version.go [hunk 1]
```diff
@@ -22,8 +22,8 @@ func newVersionCmd(rootCmd *cobra.Command) *cobra.Command {
 	// versionCmd represents the version command
 	var versionCmd = &cobra.Command{
 		Use:   "version",
-		Short: "Show Appsody CLI version",
-		Long:  ``,
+		Short: "Show Appsody CLI version.",
```
Comment (Kamran64) [inline cmd/version.go]: nit: "Output" instead of "Show"?

Comment (skoh7645) [reply cmd/version.go]: Thanks, sounds better to me

File: cmd/version.go [hunk 2]
```diff
@@ -22,8 +22,8 @@ func newVersionCmd(rootCmd *cobra.Command) *cobra.Command {
 	// versionCmd represents the version command
 	var versionCmd = &cobra.Command{
 		Use:   "version",
-		Short: "Show Appsody CLI version",
-		Long:  ``,
+		Short: "Output Appsody CLI version.",
```
Comment (neeraj-laad) [inline cmd/version.go]: Displays the version of Appsody CLI?

File: cmd/version.go [hunk 3]
```diff
@@ -22,8 +22,8 @@ func newVersionCmd(rootCmd *cobra.Command) *cobra.Command {
...
+		Long:  `Output the Appsody CLI version that is currently installed on your local system.`,
```
Comment (neeraj-laad) [inline cmd/version.go]: Displays the version of Appsody CLI that is in use?

File: cmd/version.go (hunk 2 above)
Comment (skoh7645) [reply cmd/version.go]: I prefer the word "output" over "display" in this case, but I will change the working from "Output Appsody CLI version" --> "Output the version of the Appsody CLI"

Comment (doveye) [reply cmd/version.go]: I prefer the original "show" (as in "Show the version of the Appsody CLI"), as "output" can be something generated from something else :)

Comment (skoh7645) [reply cmd/version.go]: @doveye okay, we'll go with that :)

**Pull request merged**